docs/index.md

```diff
@@ -67,12 +66,12 @@ The easiest way to get started with Swarm is to use the
 [official Docker image](https://registry.hub.docker.com/_/swarm/).
 
 ```bash
-docker pull swarm
+$ docker pull swarm
```

Comment:
So to clarify - any command WITHOUT something in it contained between < > then it should start with a $?